source/ingress.go

```diff
@@ -109,33 +110,33 @@ func (sc *ingressSource) Endpoints() ([]*endpoint.Endpoint, error) {
 		endpoints = append(endpoints, ingEndpoints...)
 	}
 
+	for _, ep := range endpoints {
+		sort.Sort(ep.Targets)
+	}
+
 	return endpoints, nil
 }
 
 // get endpoints from optional "target" annotation
 // Returns empty endpoints array if none are found.
-func getEndpointsFromTargetAnnotation(ing *v1beta1.Ingress, hostname string) []*endpoint.Endpoint {
-	var endpoints []*endpoint.Endpoint
+// FIXME(dereulenspiegel) For now we won't use this method.
+func getTargetsFromTargetAnnotation(ing *v1beta1.Ingress) endpoint.Targets {
```

Comment:
FIXME: do you use this function? 
If not delete it and have it locally stashed if you do not want to loose it.